bindings/java/pom.xml.in

```diff
@@ -1,39 +0,0 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
Was this unused? Does it make sense to use maven?

Reply:
yes, this was unused. I first tried to use maven - but maven is a piece of crab and I couldn't get it working.